.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
         stages: [commit-msg]
     -   id: check-flake8
         name: Check Flake8
-        language: script
-        entry: scripts/travisci/check_flake8.sh
+        entry: flake8
```

Comment:
is there a reason we need to add our own `system` entry, instead of just using the default pre-commit flake8 rule.

Reply:
i.e. here https://github.com/rlworkgroup/garage/pull/773/files#diff-82c3b36709926de3a1b27bea0d8258ccL70